docs/whatsnew/6.1.rst

```diff
@@ -43,6 +43,41 @@ columns with any long integers which overflowed the 32-bit integer would be retu
 as string columns. The new default behavior is consistent with ``numpy`` v2 and ``pandas``.
 
 
+Updates to `~astropy.cosmology`
+===============================
+
+:class:`~astropy.cosmology.Cosmology` as a :func:`~dataclasses.dataclass`
```

Comment:
This could have been written with less effort, but of course the effort has already been spent.
```suggestion
|Cosmology| as a :func:`~dataclasses.dataclass`
```

Reply:
✅